src/restconfig/src/test/java/org/geoserver/rest/catalog/FeatureTypeControllerTest.java

```diff
@@ -306,9 +390,12 @@ public void testGetAsXML() throws Exception {
         FeatureTypeInfo ft = catalog.getFeatureTypeByName("sf", "PrimitiveGeoFeature");
 
         /*
-         * ReferencedEnvelope re = ft.getNativeBoundingBox(); assertXpathEvaluatesTo( re.getMinX()+"" , "/featureType/nativeBoundingBox/minx", dom );
-         * assertXpathEvaluatesTo( re.getMaxX()+"" , "/featureType/nativeBoundingBox/maxx", dom ); assertXpathEvaluatesTo( re.getMinY()+"" ,
-         * "/featureType/nativeBoundingBox/miny", dom ); assertXpathEvaluatesTo( re.getMaxY()+"" , "/featureType/nativeBoundingBox/maxy", dom );
+         * ReferencedEnvelope re = ft.getNativeBoundingBox(); assertXpathEvaluatesTo( re.getMinX
```

Comment:
Do we still need this commented part?

Reply:
Good point, removed and cleared the noisy reformatting as well (the java google format plugin in intellij is not working properly after the last upgrade)